### PR TITLE
Firefox web extension

### PR DESCRIPTION
**Pull request by yodalee**
Firefox now support web extension. This pull request add a new directory of QClean of web extensions ported to Firefox.
To my opinion, since Firefox now supports web extension. The old one use Add-on SDK can become deprecated by default.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -31,6 +31,9 @@ firefox-sdk:
 	rm jetpack-sdk-latest.zip
 	mv addon-sdk-* firefox-sdk
 
+firefox-webext: QCLean-Firefox-Web-Extension
+	zip -r qclean-firefox.zip QClean-Firefox-Web-Extension
```
Comment (qcl) [inline Makefile]: Please add `rm -f qclean-firefox.zip` in `clean` section, too. Thanks!

Comment (yodalee) [inline Makefile]: Fixed

Comment (yodalee) [inline Makefile]: Wait, not merge yet.

**Comment by yodalee** [comment]
Push some typo fix commit.
If you want, I can do some squash work

**Pull request merged**